RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -9,3 +9,6 @@
     #maskFile = cms.FileInPath("RecoLocalMuon/GEMRecHit/data/maskedStrips.txt"),
     #deadFile = cms.FileInPath("RecoLocalMuon/GEMRecHit/data/deadStrips.txt")
     )
+
+from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
+phase2_GEM.toModify(gemRecHits, gemDigiLabel = cms.InputTag("simMuonGEMDigis"))
```

Comment:
```suggestion
phase2_GEM.toModify(gemRecHits, gemDigiLabel = "simMuonGEMDigis")
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

6.17 in http://cms-sw.github.io/cms_coding_rules.html#python